LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Solid
+Copyright (c) 2016 - present
```

Comment:
`- present` is not valid in a copyright statement. Also, a copyright statement should include the legal name of the entity which is claiming that copyright.